pcweb/components/footer.py

```diff
@@ -130,7 +185,7 @@ def footer(style=footer_style):
                     )
                 ),
                 rx.vstack(
-                    rx.text("Site", color="#DACEEE"),
+                    rx.text("Site", color="#E8E8F4"),
```

Comment:
Or if it's not possible, can we put the values in constants so its easier to change together